pages/settings/profile.tsx

```diff
@@ -1,12 +1,13 @@
 import { GetServerSideProps } from "next";
 import Head from "next/head";
-import { useRef, useState } from "react";
-import prisma from "../../lib/prisma";
+import {useEffect, useRef, useState} from "react";
+import prisma, {whereAndSelect} from "@lib/prisma";
```

Comment:
Codacy has a fix for the issue: [Replace `whereAndSelect` with `·whereAndSelect·`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
import prisma, { whereAndSelect } from "@lib/prisma";
```